graphql/mutation_resolver.go

```diff
@@ -847,6 +847,12 @@ func (r *mutationResolver) EditSpawnHost(ctx context.Context, spawnHost *EditSpa
 		}
 	}
 
+	if spawnHost.SleepSchedule != nil {
+		if err = h.SetSleepSchedule(ctx, *spawnHost.SleepSchedule); err != nil {
```

Comment:
SetSleepSchedule calls a validate function, which I think we wouldn't want to return internal server error for; not sure if you want to have that function call status code, or move validating out of the set function (the latter might make more sense since that's not typical).